python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_auto_materialize_asset_evaluations.py

```diff
@@ -69,6 +70,7 @@ def _create_tick(instance, status, timestamp, evaluation_id, run_requests=None,
             status=status,
             timestamp=timestamp,
             end_timestamp=end_timestamp,
+            cursor=f"{FIXED_AUTO_MATERIALIZATION_ORIGIN_ID}:{timestamp}",
```

Comment:
I think you are conflating two different kinds of cursors here

a) the cursor that the daemon uses on each tick to compute new values (that is the thing that goes in TickData)
b) the cursor that the frontend uses when paginating through a list of ticks

The "cursor" field on the Tick object as a), the "id" field on the tick object is b)

In this PR/test you are taking a b) cursor, sticking it in the TickData as a) and then querying for an a) cursor and using it for b) pagination purposes, which does not make sense to me and makes me worried this change will break b) style pagination